dtfs-central-api/src/services/state-machines/utilisation-report/event-handlers/helpers/calculate-fixed-fee.test.ts

```diff
@@ -0,0 +1,42 @@
+import { addDays } from 'date-fns';
+import { calculateFixedFee, getNumberOfDaysInCoverPeriod } from './calculate-fixed-fee';
+import { calculateFixedFeeFromDaysRemaining } from '../../../../../helpers/calculate-fixed-fee-from-days-remaining';
+
+describe('helpers/calculate-fixed-fee', () => {
+  describe('getNumberOfDaysInCoverPeriod', () => {
+    it('should return the difference in days between 2 dates', () => {
+      const difference = 12;
+
+      const today = new Date();
+      const future = addDays(today, difference);
+
+      const result = getNumberOfDaysInCoverPeriod(today, future);
+
+      expect(result).toEqual(difference);
+    });
+  });
+
+  describe('calculateFixedFee', () => {
+    it('should return the value of "calculateFixedFeeFromDaysRemaining"', () => {
+      const utilisation = 100000;
+      const interestPercentage = 5;
+      const dayCountBasis = 365;
+      const coverStartDate = new Date();
+      const coverEndDate = addDays(coverStartDate, 368);
```

Comment:
I'm curious why this is 368 and not e.g 1?

Reply:
I chose a random number

Reply:
I read this like 368 or more than 365 was required for some particular logic or something. Can we use e.g 1 instead? So the `coverEndDate` is just `coverStartDate` + 1